locale/en-us/dialog/en/another_store.dialog

```diff
@@ -0,0 +1 @@
+What store you are looking for?
```

Comment:
Should be: "What store are you looking for?"

Reply:
Fixed. Also added a newline to remove that warning, and added the newlines to the other files that lacked them.